libs/adapters/package.json

```diff
@@ -29,6 +30,7 @@
     "rollbar": "^2.6.1",
     "sequelize": "^6.32.1",
     "trpc-openapi": "1.1.2",
+    "tsx": "^4.7.2",
     "typescript-logging": "^0.6.4",
     "zod": "^3.22.4"
   },
```

Comment:
since we are already touching all package.json files, can we add exports, so we can then use --conditions to work in devmode without building?

```
"exports": {
    ".": {
      "devmode": "./src/index.ts",
      "default": "./build/index.js"
    }
  }
```

Reply:
Sorry, just seeing this. I've never mixed ts and js imports before without testing, but could we put this in a separate PR? We've had issues with new changes getting rolled into these large PRs before that have resulted in significant delays before.

Reply:
sure, let me know if you want me to take care of this after merging

Reply:
Okay, thanks, will take a look depending on timing.